flytekit/engines/unit/engine.py

```diff
@@ -107,24 +112,32 @@ def _execute_user_code(self, inputs):
                 inputs
             )
 
+    def _transform_for_user_output(self, outputs):
+        """
+        Take whatever is returned from the task execution and convert to a reasonable output for the behavior of this
+        task's unit test.
+        :param dict[Text,flytekit.models.common.FlyteIdlEntity] outputs:
+        :rtype: T
+        """
+        return outputs
+
     def register(self, identifier, version):
         raise _user_exceptions.FlyteAssertion("You cannot register unit test tasks.")
 
 
 class ReturnOutputsTask(UnitTestEngineTask):
-    def execute(self, inputs, context=None):
```

Comment:
is this no longer needed?

Reply:
It's a refactor so this method is sufficient for transforming the execute function of the super class to behave as we desire.